quarkchain/cluster/jsonrpc.py

```diff
@@ -1465,6 +1469,76 @@ def shutdown(self):
 
             await asyncio.sleep(0.5)
 
+    async def fetch_logs(self, sub_id, full_shard_id, params, websocket):
+        decoder = address_decoder
+        start_block = params.get("fromBlock", "latest")
```

Comment:
seems you can only specify topic and address, not blocks

https://infura.io/docs/ethereum/wss/eth_subscribe
![image](https://user-images.githubusercontent.com/38544039/63564630-26373880-c566-11e9-84f9-811887e7472e.png)